### DISCUSSION TITLE
MSI installer

### DISCUSSION THREAD

**Discussion by COMPsmith** [amidaware/tacticalrmm#489] [Ideas]
Any possibility on making .msi installer for GPO software deployment?

**Comment by wh1te909** [comment]
yea MSI is planned

for now you can just generate an installer exe (dynamic exe from web ui) and just use a program like orca to convert exe to msi. the dynamic exe will run unattended. the dynamic exe is generic, you can re-use it as many times as you want. so just create a very long expiry, use orca to convert it into msi and you can use that to deploy via GPO

https://theitbros.com/convert-exe-to-msi/

**Comment by Trapulo** [comment]
any news about this?
thanks

**Reply by dinger1986** [reply]
The Roadmap is here https://docs.tacticalrmm.com/roadmap/
